internal/pkg/aws/acm/acm.go

```diff
@@ -92,5 +106,24 @@ func (a *ACM) findValidAliasesAgainstCert(ctx context.Context, aliases []string,
 			validAliases = append(validAliases, alias)
 		}
 	}
-	return validAliases, nil
+	return validAliases
+}
+
+type errInValidAliasAgainstCert struct {
+	certs         []string
+	alias         string
+	domainsOfCert map[string][]string
+}
+
+func (e *errInValidAliasAgainstCert) Error() string {
+	return fmt.Sprintf("%s is not a valid domain against %s", e.alias, strings.Join(e.certs, ","))
+}
+
+func (e *errInValidAliasAgainstCert) RecommendActions() string {
+	var logMsg string
+	logMsg = fmt.Sprintf("Please use aliases that are protected by %s your imported:\n", english.Plural(len(e.certs), "certificate", ""))
+	for cert, sans := range e.domainsOfCert {
+		logMsg += fmt.Sprintf("'%s': %s\n", cert, english.WordSeries(sans[1:], ","))
```

Comment:
```suggestion
		logMsg += fmt.Sprintf("%q: %s\n", cert, english.WordSeries(sans[1:], ","))
```

Reply:
Why `sans[1:]` instead of just `sans`?

Reply:
`aws acm describe-certificate --certificate-arn` fetches me the output something like below where Domain Name is repetitive in SubjectAlternativeNames. so that is why I used `sans[1:]`.

        "CertificateArn": "arn:aws:acm:us-west-2:535307839156:certificate/c00251c7-5642-4a1e-8c19-7cf0daf92c2e",
        "DomainName": "gingo.adi.peanutsninja.com",
        "SubjectAlternativeNames": [
            "gingo.adi.peanutsninja.com",
            "gingo.bong.adi.peanutsninja.com"
        ],

Reply:
I see. From the [doc](https://docs.aws.amazon.com/acm/latest/APIReference/API_CertificateDetail.html#ACM-Type-CertificateDetail-SubjectAlternativeNames): 
> The subject alternative names include the canonical domain name (CN) of the certificate and additional domain names that can be used to connect to the website.